src/pages/Components/Flags/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Flags'
+order: 10
+---
+
+> A flag is used as a component header to categorize its content. Please use a [tag](/Components/Tags/) if you need to display additional information.
```

Comment:
Can we use multiple flags in the same element ? 
What is the difference with a tag ? What differentiate their use cases. 
Is it the main category ?

Reply:
yes you can use multiples flags at the same time. The main difference is that tags are usually not clickable